assets/controller.yaml

```diff
@@ -314,15 +314,12 @@ spec:
       volumes:
         - name: aws-credentials
           secret:
+            # The operator overwrites this on Hypershift
             secretName: ebs-cloud-credentials
         # This service account token can be used to provide identity outside the cluster.
         # For example, this token can be used with AssumeRoleWithWebIdentity to authenticate with AWS using IAM OIDC provider and STS.
         - name: bound-sa-token
-          projected:
-            sources:
-            - serviceAccountToken:
-                path: token
-                audience: openshift
+          emptyDir: {}
```

Comment:
I had to do this, otherwise it wouldn't work. @jsafrane do you think this would break standalone OCP clusters?

Reply:
I think it will break. For standalone OCP we need `projected:` volume with token provided by OCP, in HyperShift we need `emptyDir` with token provided by the minter.

Reply:
I pushed a [new commit](https://github.com/openshift/aws-ebs-csi-driver-operator/pull/159/commits/7c688277bf1767cfbc3af86ab07869b1ad751224) to revert this piece and overwrite it in Hypershift.